_sections/20-statement.md

```diff
@@ -12,33 +12,29 @@ two versions of the language for several years. While we have developed tools
 and techniques to maintain compatibility efficiently, it is a small but
 constant friction in the development of a lot of code.
 
-We are keen to use Python 3 to its full potential, and we currently accept the
-cost of writing cross-compatible code to allow a smooth transition, but we
-don’t intend to maintain this compatibility indefinitely. Although the
-transition has not been as quick as we hoped, we do see it taking place, with
+We are keen to use Python 3 to its full potential, and though 
+we had been accepting the
+cost of writing cross-compatible code to allow a smooth transition, we
+did not intend to maintain this compatibility indefinitely. Although the
+transition was not as quick as we hoped, it has taken place, with
```

Comment:
hmmm... 'has taken place' still feels a bit premature, to be honest.

Reply:
How about "it is taking place" as something stronger than "we do see it taking place" but less forceful than the past tense version.